package.json

```diff
@@ -74,6 +74,7 @@
     "@babel/core": ">=7.15.5"
   },
   "devDependencies": {
+    "@babel/plugin-proposal-decorators": "^7.16.4",
```

Comment:
This babel plugin is needed for the test I added because it loads `@babel/plugin-proposal-decorators` from the generated `.babelrc`.